hvac/v1/__init__.py

```diff
@@ -1244,10 +1247,19 @@ def _put(self, url, **kwargs):
     def _delete(self, url, **kwargs):
         return self.__request('delete', url, **kwargs)
 
-    def __request(self, method, url, headers=None, **kwargs):
+    def __request(self, method, url, **kwargs):
+        # NOTE(rswedenburg): added ability to retain the vault url's path when it's needed (for example when referencing a service path in Mesosphere)
```

Comment:
nit: There is no reason to prefix a comment with "NOTE"  (a comment is innately a note)  nor include the comment's author (that is what we use git for 😜).